File: CS/MVCSchedulerReadOnly/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using DevExpress.Web.Mvc;
using MVCSchedulerReadOnly.Models;

namespace MVCSchedulerReadOnly.Views
{
    public class HomeController : Controller
    {
        //
        // GET: /Home/
        #region #commonactions
        public ActionResult Index()
        {
            return View(SchedulerDataHelper.DataObject);
        }

        public ActionResult SchedulerPartial()
        {
            return PartialView("SchedulerPartial", SchedulerDataHelper.DataObject);
        }
        #endregion #commonactions

    }
}

File: VB/MVCSchedulerReadOnly/Controllers/HomeController.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc
Imports DevExpress.Web.Mvc
Imports MVCSchedulerReadOnly.Models

Namespace MVCSchedulerReadOnly.Views
    Public Class HomeController
        Inherits Controller

        '
        ' GET: /Home/
        #Region "#commonactions"
        Public Function Index() As ActionResult
            Return View(SchedulerDataHelper.DataObject)
        End Function

        Public Function SchedulerPartial() As ActionResult
            Return PartialView("SchedulerPartial", SchedulerDataHelper.DataObject)
        End Function
        #End Region ' #commonactions

    End Class
End Namespace
